### ISSUE TITLE
Plugins

### ISSUE THREAD

**Issue by lizzycederberg** [Threespot/aldf-wp#118]
@tedw David said he's not currently able to add plugins to the site and asked if that will change once we launch. I'm not sure what he's trying to add, but said I would ask you about it.

**Comment by tedw** [comment]
No, he won’t be able to add plugins via the admin. The only way to add plugins is by editing the code and deploying to Pantheon. This is enforced by Pantheon and is a best practice for security reasons. It also ensures admins can’t accidentally break the site by introducing a plugin that has compatibility issues with other plugins or any of the site’s core code. A developer must add the plugin and test it first before it can go into production.

**Comment by lizzycederberg** [comment]
Thanks!
